### DISCUSSION TITLE
Compiling help: error

### DISCUSSION THREAD

**Discussion by Jotaro-Gaming** [FunkinCrew/Funkin#1479] [Compiling help] [answered]
i was tryin to make an engine while i was compiling this issue apeared

obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'

**Comment by ACrazyTown** [comment]
> i was tryin to make an engine while i was compiling this issue apeared
> 
> obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'

What device are you on? From the message looks like you're on some ARM based device

**Comment by Jotaro-Gaming** [comment]
> > i was tryin to make an engine while i was compiling this issue apeared
> > obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'
> 
> What device are you on? From the message looks like you're on some ARM based device

im on a matebook 13

**Comment by ACrazyTown** [comment]
> > > i was tryin to make an engine while i was compiling this issue apeared
> > > obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'
> > 
> > 
> > What device are you on? From the message looks like you're on some ARM based device
> 
> im on a matebook 13

My bad, seems like you're trying to compile something for ARM whilst on x64, I'm pretty sure you downloaded a component on VS 2019 for ARM instead of x64

**Comment by Jotaro-Gaming** [comment]
ok
thanks

**Comment by Jotaro-Gaming** [comment]
> > > > i was tryin to make an engine while i was compiling this issue apeared
> > > > obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'
> > > 
> > > 
> > > What device are you on? From the message looks like you're on some ARM based device
> > 
> > 
> > im on a matebook 13
> 
> My bad, seems like you're trying to compile something for ARM whilst on x64, I'm pretty sure you downloaded a component on VS 2019 for ARM instead of x64

which component? I tried arm and 64

**Comment by Jotaro-Gaming** [comment]
its really starting to get on my nerves..

**Comment by AbnormalPoof** [comment]
arm64

**Comment by ACrazyTown** [comment]
> arm64

p sure funkin isn't supported on arm64, you should use x64

**Comment by napstaa967** [comment]
do "lime build windows -64" to force it to build to x64

lime doesn't support building to arm64 unless you're building to iOS or android

**Comment by Jotaro-Gaming** [comment]
k

**Comment by Jotaro-Gaming** [comment]
still shows obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'

**Comment by ACrazyTown** [comment]
> still shows obj\msvc19-arm64-nc\70e38311_Context3DElementType.obj : fatal error LNK1112: module machine type 'x64' conflicts with target machine type 'ARM64'

Are you sure you haven't accidentally installed VS2019 components for ARM?

**Comment by Jotaro-Gaming** [comment]
yes pretty sure

**Comment by Jotaro-Gaming** [comment, marked as answer]
switched computers now i can compile so ill mark this as a answer
